cmd/sync.go

```diff
@@ -422,7 +454,11 @@ type cookedSyncCmdArgs struct {
 
 	cfdMode CFDModeFlags
 
+	// lastSyncTime to find out files/Dirs changed since this time depending on CFDMode.
```

Comment:
Pls always write crisp/clear/non-confusing comments even if we have to write more, f.e.

	//
	// Time of last sync, used by the sync process.
	// This is used as a lower bound to find out files/dirs changed since last sync.
	// Depending on CFDMode this will be compared with source files' ctime/mtime or
	// the target files' ctime/mtime.
	//

Reply:
My bad.